trajectory_publisher/include/trajectory_publisher/trajectoryPublisher.h

```diff
@@ -38,9 +39,12 @@ class trajectoryPublisher
   ros::Publisher flatreferencePub_;
   ros::Publisher rawreferencePub_;
   std::vector<ros::Publisher> primitivePub_;
+
   ros::Subscriber motionselectorSub_;
   ros::Subscriber mavposeSub_;
   ros::Subscriber mavtwistSub_;
+	ros::Subscriber mavhomeposSub_;
```

Comment:
The spacing format seems to be different. Please align the spaces with the rest of the code